components/core/src/clp_s/filter/FilterReader.cpp

```
@@ -0,0 +1,109 @@
+            optional_filter_type.value(),
+            optional_filter_normalization.value(),
+            std::move(bloom_filter)
+    };
+}
+
+auto FilterReader::possibly_contains(std::string_view value) const -> bool {
+    auto const normalized_value{normalize_string(value, m_normalization)};
+    return m_bloom_filter.possibly_contains(normalized_value);
+}
+
+auto FilterReader::possibly_contains_wildcard(std::string_view value) const -> bool {
```

Cppcheck (2.20.0)

[style] The function 'possibly_contains_wildcard' is never used.

(unusedFunction)